src/marker.js

```diff
@@ -34,6 +34,10 @@ SVG.Marker = SVG.invent({
   , toString: function() {
       return 'url(#' + this.id() + ')'
     }
+  
+    // Set the orient of element
+  , orient: function(orient) {
+      return this.attr('orient',orient)
   }
```

Comment:
@askari12 You seem to be missing a `}` here. Please run `npm run build:test` before pushing, so you catch typos like this in the future.